docs/functions.md

```diff
@@ -61,4 +73,23 @@ channels:
       ...
 ```
 
-Glee maps the `onHello` operation to the `functions/onHello.js` file.
\ No newline at end of file
+Glee maps the `onHello` operation to the `functions/onHello.js` file.
+
+## Using FaaS functions as glee functions
+
+Glee can treat your FaaS function as its own by setting `operationId` to your function URL and customizing it with the `x-glee-invoke` extension:
+
+```yaml
+channels:
+  hello:
+    publish:
+      operationId: https://domain.come/path/to/your/function
+      x-glee-invoke:
+        method: get
+        headers:
+          Content-Type: application/json
+        ignoreResponse: true # By default, Glee processes the response and raises an error if it does not conform to the format specified by [functions](#functions) return type.
```

Comment:
It would be cool to note that this extension is not required. By default, we'll call the URL on `operationId` with the `POST` method, `Content-Type: application/json` header, and the message as the body.

Also, note that we can't send a request body when the method is `GET`. So we have to think what to do in this case. Do we serialize it as query params along with the URL or do we completely avoid sending the message in the body?

Reply:
Got it. I guess serialising the message makes sense here. but URLs are limited to 2048 characters which I guess can create untraceable errors (working fine for some requests and failing when the length exceeds 2048).
I think we need to respect the REST API rules/conventions here and ignore the message in GET requests.

Reply:
Yeah, I would do this too. Honestly, I think most people will use POST. If not all.